examples/android/client/app/src/main/java/flwr/android_client/MainActivity.java

```diff
@@ -127,61 +140,44 @@ public void connect(View view) {
         }
     }
 
-    public void runGRCP(View view){
-        new GrpcTask(new FlowerServiceRunnable(), channel, this).execute();
-    }
-
-    private static class GrpcTask extends AsyncTask<Void, Void, String> {
-        private final GrpcRunnable grpcRunnable;
-        private final ManagedChannel channel;
-        private final MainActivity activityReference;
-
-        GrpcTask(GrpcRunnable grpcRunnable, ManagedChannel channel, MainActivity activity) {
-            this.grpcRunnable = grpcRunnable;
-            this.channel = channel;
-            this.activityReference = activity;
-        }
-
-        @Override
-        protected String doInBackground(Void... nothing) {
-            try {
-                grpcRunnable.run(FlowerServiceGrpc.newBlockingStub(channel), FlowerServiceGrpc.newStub(channel), this.activityReference);
-                return "Connection to the FL server successful \n";
-            } catch (Exception e) {
-                StringWriter sw = new StringWriter();
-                PrintWriter pw = new PrintWriter(sw);
-                e.printStackTrace(pw);
-                pw.flush();
-                return "Failed to connect to the FL server \n" + sw;
-            }
-        }
-
-        @Override
-        protected void onPostExecute(String result) {
-            MainActivity activity = activityReference;
-            if (activity == null) {
-                return;
+    public void runGRPC(View view){
```

Comment:
We usually put just the first letter in caps (as in `GrpcRunnable`):
```suggestion
    public void runGrpc(View view){
```